Files/MultilingualResources/Files.pt-PT.xlf

```diff
@@ -6,7 +6,7 @@
     </header>
     <body>
       <group id="FILES/STRINGS/EN-US/RESOURCES.RESW" datatype="resx">
-        <trans-unit id="SideBarOpenInNewTab.Text" translate="yes" xml:space="preserve">
+            <trans-unit id="SideBarOpenInNewTab.Text" translate="yes" xml:space="preserve">
```

Comment:
Can you fix the indentation?

Reply:
You want this way?
https://github.com/files-community/Files/pull/7161/commits/db857b2889f0a1b974125f4dcbebd22e69cfff35

Reply:
Or is it with this modification?
https://github.com/files-community/Files/pull/7161/commits/ea814f427e2b428f04bf336f21a6df13d280056c